configuration/configuration.go

```diff
@@ -17,7 +17,8 @@ func (c *ConfigurationData) String() string {
 	allSettings := c.v.AllSettings()
 	y, err := yaml.Marshal(&allSettings)
 	if err != nil {
-		panic(fmt.Errorf("Failed to marshall config to string: %s", err.Error()))
+		fmt.Println(errors.Wrap(err, "Failed to marshall config to string"))
```

Comment:
Can't you make the `String()` function return `(string, error)` instead of printing the error? The caller shall decide what to do in case of an error I'd say. Sorry if my previous comment on this function was misleading.